### PR TITLE
Examples: parallax envmaps fix 

### PR DESCRIPTION
**Pull request by sneha-belkhale**
I had made a PR for the [BPCEM example](https://threejs.org/examples/?q=paralla#webgl_materials_envmaps_parallax) a few months ago, and recently found a bug. Turns out the shader code I had referenced from [this page](https://www.clicktorelease.com/blog/making-of-cruciform/) only works when the cube map is centered at the origin. 

I have fixed this issue by correctly calculating the min/max of the cube map in this PR, and it now works with any arbitrary cube map position.

### CODE REVIEW

File: examples/webgl_materials_envmaps_parallax.html [hunk 1]
```diff
@@ -356,7 +356,8 @@
 
 				boxProjectedMat.onBeforeCompile = function ( shader ) {
 
-					shader.uniforms.cubeMapSize = { value: new THREE.Vector3( 200, 100, 100 ) };
+					//these parameters are wrt the cubeCamera texture
```
Comment (mrdoob) [inline examples/webgl_materials_envmaps_parallax.html]: What's wrt?

Comment (sneha-belkhale) [reply examples/webgl_materials_envmaps_parallax.html]: with respect to*

Comment (mrdoob) [reply examples/webgl_materials_envmaps_parallax.html]: I see. Maybe better using plain English?

```js
// these parameters are for the cubeCamera texture
```

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**